kafka-minion/templates/tls-secret.yaml

```diff
@@ -1,4 +1,5 @@
-{{- if and (.Values.kafka.tls.enabled) (not .Values.kafka.tls.existingSecret) -}}
+{{- if and (.Values.kafka.tls.enabled) (.Values.kafka.tls.credentials) -}}
```

Comment:
Check if tls credentials are defined and check again below if there is no existingSecret defined @weeco.